### PR TITLE
enhance template for listing installed services

### PR DESCRIPTION
**Pull request by cpaelzer**
Suggested by @ahasenack

### CODE REVIEW

File: wiki.moinmoin [hunk 1]
```diff
@@ -187,9 +187,14 @@ RULE:   If any are present, this requires a more in-depth security review.
...
-TODO: - Package installs services, but they are safe because <TBD> (reduced
-TODO:   permissions, tmp env, restricted user, ...)
+TODO-A: - Package does not install services, timers or reoccuruing jobs
+TODO-B: - Package does install services, timers or reoccuruing jobs
+TODO-B:   <TBD> (list services, timers, jobs)
+TODO-C: - Package does install services, timers or reoccuruing jobs
+TODO-C:   <TBD> (list services, timers, jobs)
+TODO-C:   Those should be considered rather safe because <TBD> (add details like
+TODO-C:   reduced permissions, tmp env, restricted users/groups, seccomp,
+TODO-C:   apparmor, ... the more is added => more likely MIR team agrees)
```
Comment (panlinux) [inline wiki.moinmoin:197]: I would say "Package does install services, timers or recurring jobs (list) with the following security features:" then your list

File: wiki.moinmoin [hunk 2]
```diff
@@ -187,9 +187,14 @@ RULE:   If any are present, this requires a more in-depth security review.
 TODO: - no `suid` or `sgid` binaries
 TODO-A: - no executables in `/sbin` and `/usr/sbin`
 TODO-B: - Binary <TBD> in sbin is no problem because <TBD>
-TODO: - Package does not install services
-TODO: - Package installs services, but they are safe because <TBD> (reduced
-TODO:   permissions, tmp env, restricted user, ...)
+TODO-A: - Package does not install services, timers or reoccuruing jobs
```
Comment (panlinux) [inline wiki.moinmoin:190]: I think it's `recurring`?

Comment (cpaelzer) [reply wiki.moinmoin:190]: ack - fixed

File: wiki.moinmoin (hunk 1 above)
Comment (cpaelzer) [reply wiki.moinmoin:197]: Yeah I like that, it is even further improving my first try on making it clear that the `decision` isn't up to the reporter.

Comment (cpaelzer) [reply wiki.moinmoin:197]: I combined the sentence with my section, pushing the new version.

**Review by panlinux — APPROVED** [review]
+1

File: wiki.moinmoin (hunk 2 above)
Comment (slyon) [inline wiki.moinmoin:190]:
```suggestion
TODO-A: - Package does not install services, timers or recurring jobs
```

**Review by slyon — APPROVED** [review]
LGTM! But there is one more instance of "recurring" that needs fixing.

File: wiki.moinmoin (hunk 2 above)
Comment (cpaelzer) [reply wiki.moinmoin]: Fixed, thanks

**Comment by didrocks** [comment]
LGTM as well, thanks!

**Pull request closed without merging**